lib/rules/declaration-block-no-duplicate-properties/index.mjs

```diff
@@ -11,6 +10,10 @@ import ruleMessages from '../../utils/ruleMessages.mjs';
 import validateOptions from '../../utils/validateOptions.mjs';
 import vendor from '../../utils/vendor.mjs';
 
+import { isRegExp, isString } from '../../utils/validateTypes.mjs';
+import { colorProperties } from '../../reference/properties.mjs';
+import { namedColorsKeywords } from '../../reference/keywords.mjs';
```

Comment:
Let's slot these into the main block of imports. They should be alphabetical (ignoring the starting brace).